CMakeLists.txt

```diff
@@ -32,6 +32,10 @@ else (NOT BUILD_STATIC)
   set (BUILD_SHARED OFF)
 endif (NOT BUILD_STATIC)
 
+set (CMAKE_MODULE_PATH
+  ${PROJECT_SOURCE_DIR}/cmake
+)
```

Comment:
Nit: everywhere else puts the ) on the same line, and aligns multiple lines with the first arg.

Reply:
Would be nice to have an auto formatter for cmake...

Reply:
https://github.com/BlankSpruce/gersemi looks promising. Maybe we should add it to the CI

Reply:
Maybe [Gersemi](https://github.com/BlankSpruce/gersemi).

Also there's [a cmake lsp](https://github.com/neocmakelsp/neocmakelsp) that says it has a cli for code formatting.

Reply:
> https://github.com/BlankSpruce/gersemi looks promising. Maybe we should add it to the CI

Would be nice to add something.